src/app/app-routing.module.ts

```diff
@@ -37,7 +36,7 @@ const routes: Routes = [
       breadcrumb: 'Login'
     },
     component: LoginComponent,
-    canActivate: [NoWalletFoundGuard, AuthredirectGuard],
+    canActivate: [AuthredirectGuard],
```

Comment:
Instead of this guard, we always include a link to the `Create a Wallet` wizard from the login page